userdocs/profiles.dev/docs/catalog-docs/add-profiles.md

```diff
@@ -0,0 +1,104 @@
+---
+sidebar_position: 1
+---
+
+# Adding profiles
+
+:::info
+Please refer to the [Introduction](/docs/introduction#catalog) for a list of terms
+around Catalogs.
+:::
+
+To add profiles to a catalog, you must first deploy the catalog API
+and `ProfileCatalogSource` controller to your cluster.
+
+This can be done with the `pctl install` command. By default, this command
+will install latest, but you can pass a specific version with the `--version`
+flag.
+
+Once deployed, there are two methods to add profiles to the cluster catalog:
+dynamically and manually.
+
+## Dynamically populate a catalog
+
+To dynamically add all profiles and tags from profiles repositories to the catalog,
+create a `ProfileCatalogSource` providing the repositories' URLs,
+and apply the resource to the cluster.
+
+For example, the following manifest will discover all the profiles within the
+[weaveworks/profiles-examples](https://github.com/weaveworks/profiles-examples) repo
+and add an entry in the catalog for each profile tag.
+
+```yaml
+apiVersion: weave.works/v1alpha1
+kind: ProfileCatalogSource
+metadata:
+  name: nginx-catalog
+spec:
+  repositories:
+  - url: https://github.com/weaveworks/profiles-examples
+```
+
+After applying the manifest and waiting a moment, we can use `pctl search` to see the
+catalogued profiles:
```

Comment:
probably worth adding an example that requires a secret 

config map: https://github.com/weaveworks/profiles/blob/c5964145ad61ed255dc15ca3467b90239aa765e5/tests/acceptance/acceptance_test.go#L340-L353

catalog example: https://github.com/weaveworks/profiles/blob/c5964145ad61ed255dc15ca3467b90239aa765e5/tests/acceptance/acceptance_test.go#L358-L378

Reply:
This is for adding private repositories yes?

Reply:
Yep